adaptive_ai/adaptive_ai/core/config.py

```
@@ -101,101 +97,33 @@ class HealthConfig(BaseModel):
     timeout: float = 5.0
 
 
+# NEW: Added EmbeddingCacheSettings
+class EmbeddingCacheSettings(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)